lib/index.js

```diff
@@ -39,50 +44,28 @@ AdWords.ensure(function(msg, settings){
  * https://developers.google.com/app-conversion-tracking/ios/conversion-tracking-server
  * https://developers.google.com/app-conversion-tracking/android/conversion-tracking-server
  *
- * @param {Track} msg
- * @param {Function} callback
+ * @param {Track} payload
+ * @param {Function} done
  */
 
-AdWords.prototype.track = function(msg, callback){
-  var events = this.settings.events;
-  var payload = mapper.track(msg, this.settings);
+AdWords.prototype.track = function(payload, done){
```

Comment:
It might be helpful to keep this named `callback` (or `fn`) so its not confused with the `done` callback inside the `each` below